sources/PseudoTerminal.m

```diff
@@ -326,11 +326,13 @@ + (NSInteger)styleMaskForWindowType:(iTermWindowType)windowType {
         case WINDOW_TYPE_BOTTOM_PARTIAL:
         case WINDOW_TYPE_LEFT_PARTIAL:
         case WINDOW_TYPE_RIGHT_PARTIAL:
-        case WINDOW_TYPE_NO_TITLE_BAR:
             return NSBorderlessWindowMask | NSResizableWindowMask;
 
         case WINDOW_TYPE_TRADITIONAL_FULL_SCREEN:
             return NSBorderlessWindowMask;
+            
+        case WINDOW_TYPE_NO_TITLE_BAR_ROUNDED:
+            return NSTitledWindowMask | NSFullSizeContentViewWindowMask | NSMiniaturizableWindowMask | NSResizableWindowMask;
```

Comment:
Yech, I hadn't noticed that NSFullSizeContentViewWindowMask was new to 10.10. We'll have to make this conditional on OS version. I'm going to put this PR on hold for the time being because El Capitan is going to make my life a little crazy and I don't want to introduce another variable--but I do promise to come back to it.